app/assets/stylesheets/locals/search_form.css.scss

```diff
@@ -1,19 +1,17 @@
 // Default style for search forms
 .search_form {
 
-  // Default width for anything less than large screen
-  width: 320px;
+  width: 100%;
 
   form {
 
     @include flexbox();
     @include flexbox-center-vert();
 
-    width: 100%;
     input {
       background-color: $almost-white;
       color: #000;
-      width: 90% !important;
+      width: 100% !important;
```

Comment:
@mccalluc Can you confirm this in other browsers and widths? I think the 90% for the `width` on `.search_form form input` was to leave room for the magnifying glass submit button. I'd be worried that changing it to 100% may have side effects.

If it works at all breakpoints in FF, Chrome, and Safari, then :+1: from me. Go ahead and merge.